### PR TITLE
fix: typos

### PR DESCRIPTION
**Pull request by n1c01a5**

### CODE REVIEW

**Comment by cameel** [comment]
Hello @n1c01a5. I'm currently converting this repository to make it usable by the new translation bot (see [Introducing the Translations PR Bot!](https://forum.soliditylang.org/t/introducing-the-translations-pr-bot/1115)). As a part of this process I switched the main branch of the repository to `develop`, which now has full history.

To ensure that open PRs will get merged correctly after this change, a rebase is needed. I just did this for you but unfortunately github is a bit picky with such things and did not let me change the base branch, which is necessary as well. Instead it closed it and since it's an external PR, I cannot reopen it. I could create a fresh PR but then you would not be able to push anything to it.

In short, please create a new PR with the same branch. The branch was correctly updated and it's just a problem with the PR itself.
